tslearn/docs/examples/plot_dtw.py

```diff
@@ -55,7 +66,7 @@
 ax_s_x.axis("off")
 ax_s_x.set_xlim((0, sz - 1))
 
-ax_s_y.plot(- dataset_scaled[0], numpy.arange(sz)[::-1], "b-", linewidth=3.)
+ax_s_y.plot(- dataset_scaled[0], numpy.arange(sz), "b-", linewidth=3.)
```

Comment:
Are you sure of that? I mean, are you sure that the time series will be plotted in the right order wrt the cost matrix?

Reply:
I changed the origin for the image of the cost matrix (see line 59), it is not `'lower'` instead of the default value (`'upper'`). So it looks like this now (which will be more familiar for users, since the cost matrix is usually presented this way in the papers, i.e. going from bottom left to top right):
![sphx_glr_plot_dtw_001](https://user-images.githubusercontent.com/17053934/78360415-0fa29680-75b7-11ea-8984-e072d8dc980e.png)

Reply:
OK, did not see that.